Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -452,7 +456,7 @@ private void InitializeServiceLocator()
                 }
             }
 
-            #endregion Service Registration
+#endregion Service Registration
```

Comment:
Looks like some whitespace changes here. We've typically had `#region`s indented as they were before this change.

Reply:
ah gotcha, that was on purpose but I'll switch back